modules/apps/portal-search/portal-search-api/bnd.bnd

```diff
@@ -9,6 +9,7 @@ Export-Package:\
 	com.liferay.portal.search.analysis,\
 	com.liferay.portal.search.asset,\
 	com.liferay.portal.search.batch,\
+	com.liferay.portal.search.blueprint,\
```

Comment:
i think "experiences" would probably be a more consistent name for the package than "blueprint"

Reply:
and gw baseline needs to be ran on the modules to update semver info